src/api/subscriptions.js

```diff
@@ -351,9 +354,61 @@ export class Subscriptions {
    * @return {?}
    */
   setPublisherProvidedId(publisherProvidedId) {}
+
+  /**
+   * Starts the intervention audience action flow.
```

Comment:
Could this say more about what interventions are, and when publishers might want to call the method?

Reply:
Done.

Reply:
Thanks!